src/tsconfig.spec.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think you can remove `tsconfig.test.json` in the root as it is not used anymore.